### PR TITLE
Small space bar mapping additions

### PR DESCRIPTION
**Pull request by Doxxmedearly**
Small tweaks. Mostly to add the long-range holopad so they have a way to reach out to nearby ships.
Also added:
Another light to the backroom.
Crayons for making drinks.
A new area for the hangar.
And made the base_turf into plating in case any deconstruction happens.

### CODE REVIEW

**Comment by Doxxmedearly** [comment]
!review

File: maps/away/generic/space_bar.dm [hunk 1]
```diff
@@ -6,6 +6,7 @@
 	spawn_weight = 1
 	spawn_cost = 2
 	id = "space_bar"
+	template_flags = TEMPLATE_FLAG_SPAWN_GUARANTEED
```
Comment (SleepyGemmy) [inline maps/away/generic/space_bar.dm:9]: intended?

File: maps/away/generic/space_bar.dm [hunk 2]
```diff
@@ -20,5 +21,9 @@
...
 	flags = RAD_SHIELDED
 	requires_power = FALSE
-	base_turf = /turf/space
-	no_light_control = TRUE
\ No newline at end of file
+	base_turf = /turf/simulated/floor/plating
+	no_light_control = TRUE
+
+/area/space_bar/hangar
+	name = "Spacer Bar Hangar"
```
Comment (SleepyGemmy) [inline maps/away/generic/space_bar.dm:28]: typo.

File: maps/away/generic/space_bar.dm (hunk 1 above)
Comment (Doxxmedearly) [reply maps/away/generic/space_bar.dm:9]: Hahahahahahahahahahahaha i dumb

File: maps/away/generic/space_bar.dm (hunk 2 above)
Comment (Doxxmedearly) [reply maps/away/generic/space_bar.dm:27]: No, the area for the space bar is called the Spacer Bar, so that is consistent.

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

**Pull request merged**